redis-cli.go

```diff
@@ -41,6 +42,12 @@ const (
 func main() {
 	flag.Parse()
 
+	portInt, err := strconv.Atoi(*portStr)
+	if err != nil {
+		fmt.Printf("(error) %s", err.Error())
+	}
+	port = &portInt
```

Comment:
how about use portStr directly? 
https://github.com/holys/redis-cli/blob/88d832719fa6e915be4438bbdc686936338396d1/redis-cli.go#L275

Reply:
Addressed in commit https://github.com/holys/redis-cli/pull/9/commits/0b0d2b2edf84ea5121a30b19a2147ae01de51856.